Submitted By: Patches Project <patches at linuxfromscratch dot org>
Date: 2003-07-16
Initial Package Version: 1.0
Upstream Status: In CVS
Origin: http://patches.linuxfromscratch.org
Description: This is a sample patch. This was created so that there is an
             easy reference for those submitting patches.

diff -Naur package-1.0.orig/README package-1.0/README
--- package-1.0.orig/README     2003-08-17 06:25:37.000000000 +0000
+++ package-1.0/README  2003-08-17 06:25:56.000000000 +0000
@@ -1,4 +1,3 @@
 Test file for patches project
-Who created this silly file.
 What is this new package package?
 Already at Version 1.0? Fast paced!!!
